product_docs/docs/postgres_distributed_for_kubernetes/1/known_issues.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title: 'Known issues and limitations'
+---
+
+These known issues and limitations are in the current release of EDB Postgres Distributed for Kubernetes. 
+
+## Upgrades 
+
+This version of EDB Postgres Distributed for Kubernetes does **not support** migrating from existing Postgres databases, nor upgrading 
```

Comment:
does **not support**
sounds strange to me. I'd propose either of:
- **does not support**
- does **not** support